tests/core/v5_1/test_dispatcher_session_management.py

```diff
@@ -5,58 +5,25 @@
 
 
 @pytest.mark.trio
-async def test_dispatcher_detects_handshake_timeout_as_initiator(
-    alice, bob, autojump_clock
-):
-    alice.enr_db.set_enr(bob.enr)
-    bob.enr_db.set_enr(alice.enr)
-
-    async with alice.client() as alice_client:
-        async with bob.client() as bob_client:
-            async with alice.events.session_created.subscribe_and_wait():
-                async with bob.events.packet_sent.subscribe() as subscription:
-                    await alice_client.send_ping(
-                        endpoint=bob.endpoint, node_id=bob.node_id
-                    )
-                    _, envelope = await subscription.receive()
-                    assert envelope.packet.is_who_are_you
-                    bob_client.get_manager().cancel()
-        # Here we know that the session with bob was created and that bob has terminated
-        #
-        # Now we wait for the timeout to be triggered
-        with trio.fail_after(SESSION_IDLE_TIMEOUT + 1):
-            async with alice.events.session_timeout.subscribe() as subscription:
-                session = await subscription.receive()
-                assert session.remote_node_id == bob.node_id
-
-
-@pytest.mark.trio
-async def test_dispatcher_detects_handshake_timeout_as_recipient(
-    alice, bob, autojump_clock
+async def test_dispatcher_intiator_and_recipient_detects_handshake_timeout(
```

Comment:
@pipermerriam If you don't mind giving one final :eyes: to this test change and double check that I'm not losing any test coverage with this change.